share/spice/drupal/drupal.js

```diff
@@ -0,0 +1,78 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_drupal = function(api_result){
+
+        // Validate the response
+        if (!api_result || api_result.list === "") {
```

Comment:
Check if `api_result.list` is an empty array instead of checking if it's an empty string. `[] === ""` will result to `false`.